cluster/gce/config-default.sh

```diff
@@ -314,28 +315,29 @@ fi
 #   new subnetwork will be created for the cluster.
 ENABLE_IP_ALIASES=${KUBE_GCE_ENABLE_IP_ALIASES:-false}
 NODE_IPAM_MODE=${KUBE_GCE_NODE_IPAM_MODE:-RangeAllocator}
-if [ ${ENABLE_IP_ALIASES} = true ]; then
+if [ "${ENABLE_IP_ALIASES}" = true ]; then
```

Comment:
shellcheck said SC2086. `ENABLE_IP_ALIASES` can be `true` or `false`, so it is no problem to add double quote.
```
if [ ${ENABLE_IP_ALIASES} = true ]; then
     ^------------------^ SC2086: Double quote to prevent globbing and word splitting.
```